packages/storage-plus/src/de.rs

```diff
@@ -191,18 +139,13 @@ impl<T: KeyDeserialize, U: KeyDeserialize> KeyDeserialize for (T, U) {
 
         Ok((T::from_slice(t)?, U::from_slice(u)?))
```

Comment:
hmmm.. could you use `from_vec` here as well?

I guess you would need [value.split_off](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.split_off) rather than `split_at`. But should be more consistent/efficient